net.sf.eclipsecs.checkstyle/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="checkstyle-8.35-all.jar" sourcepath="checkstyle-checkstyle-8.35.zip"/>
+	<classpathentry exported="true" kind="lib" path="checkstyle-8.36-all.jar" sourcepath="checkstyle-checkstyle-8.36.zip"/>
```

Comment:
8.36.1 should be used.
https://checkstyle.org/releasenotes.html#Release_8.36.1

Reply:
Done